tools/run_tests/dockerize/docker_run_tests.sh

```diff
@@ -42,7 +42,21 @@ export PATH=$PATH:/usr/bin/llvm-symbolizer
 chown $(whoami) $XDG_CACHE_HOME
 
 mkdir -p /var/local/git
-git clone --recursive /var/local/jenkins/grpc /var/local/git/grpc
+git clone /var/local/jenkins/grpc /var/local/git/grpc
+
+# (todo (mattkwong): the /var/local/jenkins/grpc has no submodules and boringssl 
```

Comment:
> where did you see "has no submodules and boringssl has non-submodule files in it"?

This was an issue when I was testing locally on my machine, but I have fixed it - should be unrelated to Jenkins.

The issues I saw with Jenkins that I was investigating was a portability test from a couple days ago: [https://grpc-testing.appspot.com/job/gppr/7281/](https://grpc-testing.appspot.com/job/gppr/7281/). The issue with the 4 failing gcc tests is:

Cloning into '/var/local/git/grpc'...
done.
Entering 'third_party/boringssl'
Submodule 'third_party/boringssl' (https://github.com/google/boringssl.git) registered for path 'third_party/boringssl'
fatal: reference repository '/var/local/jenkins/grpc/third_party/boringssl' is not a local directory.
Cloning into 'third_party/boringssl'...
Clone of 'https://github.com/google/boringssl.git' into submodule path 'third_party/boringssl' failed
Stopping at 'third_party/boringssl'; script returned non-zero status.

I'm currently trying to replicate this in Jenkins, if it will let me, so I can fix it.